src/ncp/ncp_base.cpp

```diff
@@ -1829,21 +1830,48 @@ ThreadError NcpBase::GetPropertyHandler_THREAD_CHILD_TABLE(uint8_t header, spine
     {
         if (childInfo.mTimeout > 0)
         {
+            modeFlags = 0;
+
+            if (childInfo.mRxOnWhenIdle)
+            {
+                modeFlags |= kThreadModeTLV_Receiver;
```

Comment:
Good idea. I used the existing constants.
Will change them (in a separate PR - which we can push in before this and then rebase)...